$OpenBSD: patch-build_detectsys_py,v 1.4 2012/12/23 03:25:18 bentley Exp $
Give this a chance to build on sgi.

Committed upstream in r13070.
--- build/detectsys.py.orig	Sat Dec 22 20:18:26 2012
+++ build/detectsys.py	Sat Dec 22 20:18:41 2012
@@ -25,7 +25,7 @@ def detectCPU():
 		return 'ppc64' if cpu.endswith('64') else 'ppc'
 	elif cpu.startswith('arm'):
 		return 'arm'
-	elif cpu.startswith('mips'):
+	elif cpu.startswith('mips') or cpu == 'sgi':
 		return 'mipsel' if cpu.endswith('el') else 'mips'
 	elif cpu == 'm68k':
 		return 'm68k'
